Samples/Sample.Xamarin.Core/Sample.Xamarin.Core.csproj

```diff
@@ -1,11 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="Directory.Build.props" />
-	
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
+    <!-- Fixes error CSC: error CS0006 Metadata file could not be found for the VS4MAC when doing a release build. -->
```

Comment:
Where is the `VS4MAC` "dependency" coming from? Is this for local development?

Reply:
it happens only on the iOS sample, on a release build, this project isn't copied to the iOS project and the mentioned error happens.
I applied the "fix" mentioned here and so far the Sample.Xamarin.Core is being now properly  linked to the iOS sample project.
https://github.com/dotnet/msbuild/issues/7355#issuecomment-1025921050